diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 24afa3db42..d3f04aed2d 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -130,7 +130,6 @@ if(WIN32 AND NOT MINGW)
   set(CMAKE_RC_FLAGS
       "${CMAKE_RC_FLAGS} -D DUCKDB_DEV_ITERATION=\"${RC_DEV_ITERATION}\"")
 
-  target_sources(duckdb PRIVATE version.rc)
 endif()
 
 install(
diff --git a/tools/shell/CMakeLists.txt b/tools/shell/CMakeLists.txt
index 0d26359008..04c5c7f01f 100644
--- a/tools/shell/CMakeLists.txt
+++ b/tools/shell/CMakeLists.txt
@@ -77,7 +77,6 @@ if(WIN32 AND NOT MINGW)
       "${CMAKE_RC_FLAGS} -D DUCKDB_DEV_ITERATION=\"${RC_DEV_ITERATION}\"")
   set(CMAKE_RC_FLAGS
       "${CMAKE_RC_FLAGS} -D DUCKDB_COPYRIGHT_YEAR=\"${DUCKDB_COPYRIGHT_YEAR}\"")
-  target_sources(shell PRIVATE rc/duckdb.rc)
 endif()
 
 set_target_properties(shell PROPERTIES OUTPUT_NAME duckdb)
